test/extensions/formatter/cel/cel_test.cc

```diff
@@ -0,0 +1,133 @@
+#include "envoy/config/core/v3/substitution_format_string.pb.validate.h"
+
+#include "source/common/formatter/substitution_format_string.h"
+#include "source/common/formatter/substitution_formatter.h"
+#include "source/extensions/formatter/cel/cel.h"
+
+#include "test/mocks/server/factory_context.h"
+#include "test/mocks/stream_info/mocks.h"
+#include "test/test_common/utility.h"
+
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace Formatter {
+
+class CELFormatterTest : public ::testing::Test {
+public:
+  Http::TestRequestHeaderMapImpl request_headers_{
+      {":method", "GET"},
+      {":path", "/request/path?secret=parameter"},
+      {"x-envoy-original-path", "/original/path?secret=parameter"}};
+  Http::TestResponseHeaderMapImpl response_headers_;
+  Http::TestResponseTrailerMapImpl response_trailers_;
+  StreamInfo::MockStreamInfo stream_info_;
+  std::string body_;
+
+  envoy::config::core::v3::SubstitutionFormatString config_;
+  NiceMock<Server::Configuration::MockFactoryContext> context_;
+};
+
+#ifndef WIN32 // CEL is not supported on Windows
```

Comment:
CEL is supported, we just can't build ANTLR :) Please replace with USE_CEL_PARSER